conf/defaults.yaml

```diff
@@ -285,3 +285,6 @@ pacemaker.kerberos.users: []
 #default storm daemon metrics reporter plugins
 storm.daemon.metrics.reporter.plugins:
      - "org.apache.storm.daemon.metrics.reporters.JmxPreparableReporter"
```

Comment:
can't we piggyback on codahale to report these metrics? We can have one more config
```storm.cluster.metrics.reporter.plugins:
   - "org.apache.storm.daemon.metrics.reporters.JmxPreparableReporter"
```
I haven't gone through diff in nimbus completely but I suppose they can be modeled as gauages. It would be lot more simplified in my opinion.